changelog.txt

```diff
@@ -7,6 +7,7 @@ version 1.1.2 -
 new features:
 - Added the ability to force polarization to be only eTheta or ePhi in the reconstruction of the electric field using voltageToEfieldConverter.py
 - new detector which allows to change certain parameters of the detector description to study systematic uncertainties.
+- Simulations with alias (higher Nyquist zones) now supported
```

Comment:
specify that this only applies to the phased array trigger simulator (I guess)

Reply:
Changed.